agent/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apk update && apk --update add tzdata ruby ruby-irb ruby-bigdecimal \
 ADD Gemfile /app/
 ADD Gemfile.lock /app/
 
-RUN apk --update add --virtual build-dependencies ruby-dev build-base openssl-dev && \
+RUN apk --update add --virtual build-dependencies ruby-dev build-base openssl-dev git && \
```

Comment:
The `git` is required for `bundle install` when testing changes to `kontena-websocket-client` using gems from git.